src/methods/user/user.js

```diff
@@ -0,0 +1,21 @@
+// Note this package should be removed from package.json when
+// this temporary code is moved to backend
+import sign from 'jwt-encode';
+
+export const login = (email, password) => {
```

Comment:
why not use async here?

Reply:
Is possible sure, this function is still very temporary though. Likely completely replaced when dealing with an actual API ;)